pyproject.toml

```diff
@@ -173,6 +174,21 @@ requires = ["setuptools>=77.0.0",
             "pyerfa>=2.0.1.1"]
 build-backend = "setuptools.build_meta"
 
+[dependency-groups]
```

Comment:
@neutrinoceros , does the dev docs need updating to teach devs how to use this locally?